<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMFgui</class>
 <widget class="QDialog" name="DMFgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMFgui</string>
  </property>
  <widget class="QTabWidget" name="tabs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>DMF</string>
    </attribute>
    <widget class="QGroupBox" name="commandGroup">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>0</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Command</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="mixButton">
        <property name="text">
         <string>Mix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sequenceButton">
        <property name="text">
         <string>Sequence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="splitButton">
        <property name="text">
         <string>Split</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="SerialMonitorGroup">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>441</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Serial Monitor</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="UndoButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Undo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>401</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>DMF Device</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="enterButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Map</string>
     </property>
    </widget>
    <widget class="QPushButton" name="autogen_Button">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>auto generate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="writeButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Write</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ReadButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="WritetoTextBox">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>350</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>WritetoText</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PreviewSpeedEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>350</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Preview">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>350</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Preview Speed (secs)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PreviewButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="realTimeBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Real-time sequencing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="readCSVButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ReadCSV</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fillTextButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>FillText</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newLayoutButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>New Layout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadLayoutButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Layout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveLayoutButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Layout</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="DeleteBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Buttons</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Function Generator</string>
    </attribute>
    <widget class="QPushButton" name="Voltage_SendButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Voltage:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Nemesys</string>
    </attribute>
    <widget class="QComboBox" name="unitsComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume Units:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="doseButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dose</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>171</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Target Values</string>
     </property>
     <widget class="QRadioButton" name="targetVolume">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Volume</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="targetFlow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Flow rate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="targetVolumeEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="targetFlowRateEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="refillButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refill</string>
     </property>
    </widget>
    <widget class="QPushButton" name="emptyButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Empty</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Current Values Dosing Unit 1</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Volume:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Flow rate:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Syringe Level:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TargetMonitorFlowRate">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TargetMonitorSyringeLevel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="OpenButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open NemeSYS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close NemeSYS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Flow rate Units</string>
     </property>
    </widget>
    <widget class="QComboBox" name="funitscomboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>69</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dosing Units</string>
     </property>
    </widget>
    <widget class="QComboBox" name="dosingUnitscomboBox">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="CalibrateButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibrate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StopButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="MButton1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>MButton</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="MonitorBox1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MonitorBox</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="targetRefreshRateEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh Rate (ms):</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Current Values Dosing Unit 2</string>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Volume:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Flow rate:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Syringe Level:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TargetMonitorFlowRate2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TargetMonitorSyringeLevel2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>60</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Current Values Dosing Unit 3</string>
     </property>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Volume:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Flow rate:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Syringe Level:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TargetMonitorFlowRate3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="TargetMonitorSyringeLevel3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="MButton2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>MButton</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="MonitorBox2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>190</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MonitorBox</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="MButton3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>210</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>MButton</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="MonitorBox3">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>190</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MonitorBox</string>
     </property>
    </widget>
    <zorder>MButton1</zorder>
    <zorder>unitsComboBox</zorder>
    <zorder>label_7</zorder>
    <zorder>doseButton</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>refillButton</zorder>
    <zorder>emptyButton</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>OpenButton</zorder>
    <zorder>CloseButton</zorder>
    <zorder>label_10</zorder>
    <zorder>funitscomboBox</zorder>
    <zorder>label_11</zorder>
    <zorder>dosingUnitscomboBox</zorder>
    <zorder>CalibrateButton</zorder>
    <zorder>StopButton</zorder>
    <zorder>MonitorBox1</zorder>
    <zorder>targetRefreshRateEdit</zorder>
    <zorder>label_12</zorder>
    <zorder>groupBox_4</zorder>
    <zorder>groupBox_5</zorder>
    <zorder>MButton2</zorder>
    <zorder>MonitorBox2</zorder>
    <zorder>MButton3</zorder>
    <zorder>MonitorBox3</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
